src/main/java/dev/monogon/cue/lang/psi/impl/CuePsiImplUtil.java

```diff
@@ -37,4 +42,45 @@ public static boolean isOptionalFieldName(@NotNull CueLabelName name) {
         var next = name.getNextSibling();
         return next != null && next.getNode().getElementType() == CueTypes.QMARK;
     }
+
+    static Logger logger = Logger.getInstance(CuePsiImplUtil.class);
+
+    public static ItemPresentation getPresentation(CueField field) {
+//        logger.warn("getting representation for "+ field.getName());
+        var name = field.getLabelList().stream().map(PsiElement::getText).reduce((x,y) -> x + "." + y);
```

Comment:
You could `return null` here if name is empty, because the presentation would have no presentable text in this case.